### PR TITLE
Add `stop_prank` hint local

### LINKED ISSUE #1484: Implement `stop_prank` hint local
Related: #1471 
_Note_: Should use `cancel_prank`

### PR DESCRIPTION
**Pull request by cptartur**
Closes #1484 

Related to https://github.com/software-mansion-labs/cairo/pull/35

### CODE REVIEW

File: tests/integration/cairo1_compiler_bindings/library_functions_test/stop_prank_test.cairo
Comment (karol-bisztyga) [inline tests/integration/cairo1_compiler_bindings/library_functions_test/stop_prank_test.cairo:4]:
```suggestion
      Result::Ok(_) => (),
```
Isn't this `()`?

**Comment by karol-bisztyga** [comment]
Let's wait until the cairo pr is merged and ci passes

File: tests/integration/cairo1_compiler_bindings/library_functions_test/stop_prank_test.cairo
Comment (cptartur) [reply tests/integration/cairo1_compiler_bindings/library_functions_test/stop_prank_test.cairo:4]: It fails with just `()`, `(_)` is okay.

**Review by karol-bisztyga — APPROVED** [review]

**Review by Arcticae — APPROVED** [review]

**Pull request merged**
